packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -35,6 +38,14 @@ AssertExists() {
   return 0
 }
 
+PlistHasField() {
+  if /usr/libexec/PlistBuddy -c "Print :$1" $2 1>/dev/null 2>/dev/null; then
```

Comment:
You should be able to use `CMD > /dev/null 2>&1` for this check. 

Here and elsewhere: Since this is bash consider using a double-square bracket test.

Reply:
Changed the redirect, I don't think double bracket is what I want here since I'm checking the return code of the process.